src/solver/optimisation/opt_pilotage_optimisation_lineaire.cpp

```diff
@@ -53,7 +53,7 @@ bool OPT_PilotageOptimisationLineaire(PROBLEME_HEBDO* problemeHebdo, uint numSpa
             problemeHebdo->NombreDeJours = (int)(problemeHebdo->NombreDePasDeTemps
                                                  / problemeHebdo->NombreDePasDeTempsDUneJournee);
 
-            if (problemeHebdo->OptimisationAuPasHebdomadaire == NON_ANTARES)
+            if (!problemeHebdo->OptimisationAuPasHebdomadaire)
```

Comment:
We should make **OptimisationAuPasHebdomadaire** a **bool**, but I suppose it's already planned. Is it ?

Reply:
Yes will be done in a different PR 
https://github.com/AntaresSimulatorTeam/Antares_Simulator/issues/1144